app/src/main/java/com/example/photofy/RecommendationsService.java

```diff
@@ -35,16 +30,17 @@ public class RecommendationsService {
     private StringBuilder endpoint = new StringBuilder("https://api.spotify.com/v1/recommendations?market=US");
     private List<Song> songs = new ArrayList<>();
 
+    private Context context;
     private String genre;
     private final SharedPreferences sharedPreferences;
     private final RequestQueue queue;
 
     public RecommendationsService(Context context, String genre) {
```

Comment:
no need to pass context as service needs not be aware it is running under a context. this service is not android specific and does not interact with the UI directly.